keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -64,6 +67,7 @@ class DeliveryConfigController(
     consumes = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE],
     produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
+  @PreAuthorize("@authorizationSupport.userCan('WRITE', 'DELIVERY_CONFIG', #deliveryConfig.name)")
```

Comment:
this is just a get call, although it does do calculations on the back end. Why did you chose 'WRITE' permissions here?

Reply:
Copy/paste mistake. Will fix.